src/tools/kdb/export.cpp

```diff
@@ -34,6 +34,11 @@ int ExportCommand::execute (Cmdline const & cl)
 
 	Key root = cl.createKey (0);
 
+	//	if (root.getNamespace() == kdb::ElektraNamespace::CASCADING) {
```

Comment:
Don't forget to either remove this code or uncomment it, if you need it here.

Reply:
Missed it, thanks